src/Tester/BasicActivationTests.cs

```diff
@@ -15,36 +14,26 @@
 namespace UnitTests.General
 {
     [TestClass]
-    public class BasicActivationTests : UnitTestSiloHost
+    public class BasicActivationTests : UnitTestSiloHostEnsureDefaultStarted3
     {
-
-        public BasicActivationTests()
-            : base(new TestingSiloOptions { StartFreshOrleans = true })
-        {
-        }
-
-        [ClassCleanup]
-        public static void MyClassCleanup()
-        {
-            StopAllSilos();
-        }
-
         [TestMethod, TestCategory("Functional"), TestCategory("ActivateDeactivate"), TestCategory("GetGrain")]
         public void BasicActivation_ActivateAndUpdate()
         {
-            ITestGrain g1 = GrainClient.GrainFactory.GetGrain<ITestGrain>(1);
-            ITestGrain g2 = GrainClient.GrainFactory.GetGrain<ITestGrain>(2);
-            Assert.AreEqual(1L, g1.GetPrimaryKeyLong());
-            Assert.AreEqual(1L, g1.GetKey().Result);
-            Assert.AreEqual("1", g1.GetLabel().Result);
-            Assert.AreEqual(2L, g2.GetKey().Result);
-            Assert.AreEqual("2", g2.GetLabel().Result);
+            long g1Key = GetRandomGrainId();
+            long g2Key = GetRandomGrainId();
```

Comment:
Some parts of this change set like this could be split into separate PR because I don't think they have a direct dependency on the no-restart part of the change? 
This test should work even under the current TestingSiloHost implementation, won't it?